main.coffee

```diff
@@ -179,6 +179,23 @@ americano._loadPlugins = (options, app, callback) ->
         callback()
 
 
+# Listen for http (or https) connections
+americano._listen = (app, options, callback) ->
+    if options.ssl
+        https = require 'https'
+        crtPath = process.env.SSL_CRT_PATH or '/etc/cozy/server.crt'
+        keyPath = process.env.SSL_KEY_PATH or '/etc/cozy/server.key'
+        tlsOptions =
+            cert: fs.readFileSync crtPath
+            key:  fs.readFileSync keyPath
```

Comment:
They are other options which might prove useful for americano user, may be we can accept a options.tls.xxx object, extends it with the readFiles and use that instead